packages/headless/src/app/engine-configuration.ts

```diff
@@ -50,6 +50,11 @@ export interface EngineConfiguration<
    * The access token to use to authenticate requests against the Coveo Cloud endpoints. Typically, this will be an API key or search token that grants the privileges to execute queries and push usage analytics data in the target Coveo Cloud organization.
    */
   accessToken: string;
+  /**
```

Comment:
This cannot be added to the "global engine config". 

This is the engine config that every Coveo use case supported by Headless will inherit from. 

This config needs to be colocated to where to only where this is actually needed. 

In other words, this PR should not be touching any files related to `engine-configuration`, `engine.ts`, and possibly `configuration` (TBD...). 

This could possibly be solely located in the existing question answering slice ?

Reply:
I get your very relevant concern.
Down the road I think that all the engines that use RGA will leverage this Answer configuration ID, but this doesn't means we absolutely need to modify the core interfaces. Or the core engine. I'll see what I can do.

Reply:
I removed the change I made to `engine-configuration` and and `engine`. But kept the ones in configuration-slice and configuration-actions

Reply:
> This could possibly be solely located in the existing question answering slice ?

Since the usage of the `answerConfiguration` will influence components outside the question answering, like the searchBox, and to ease to configuration of the interfaces using the answer manager. I think we have more advantage to set its entry point at the engine initialization level.